projects/services/storage-service-typescript/src/server.ts

```diff
@@ -0,0 +1,28 @@
+
+import * as path from "path";
+
+import { grpcServer, topologyPeers } from "orbs-core-library";
+import BlockStorageService from "./block-storage-service";
+import StateStorageService from "./state-storage-service";
+
+export default function(nodeTopology: any, env: any) {
+    const { NODE_NAME, NODE_ENV, BLOCK_STORAGE_POLL_INTERVAL, BLOCK_STORAGE_DB_PATH } = env;
+
+    if (!NODE_NAME) {
+        throw new Error("NODE_NAME can't be empty!");
+    }
+
+    const peers = topologyPeers(nodeTopology.peers);
+    // TODO: this is very fragile - refactor to make this less hardcoded
+    const blockStorageDBPath = BLOCK_STORAGE_DB_PATH || path.resolve(path.join("../../../db", NODE_ENV || "development") + "blocks.db");
+    const blockStorageConfig = {
+      nodeName: NODE_NAME,
+      dbPath: blockStorageDBPath,
+      pollInterval: Number(BLOCK_STORAGE_POLL_INTERVAL) || 5000
```

Comment:
It's interesting that block storage poll interval is 5 seconds while we produces a block every 0.5 seconds.